ebb_lint/test/test_flake8.py

```diff
@@ -1,13 +1,15 @@
+# -*- coding: utf-8; -*-
 from __future__ import unicode_literals
 
 import ast
 import functools
+import pytest
 import re
+import six
 import sys
 
-import pytest
-import six
 from flake8.engine import get_parser
+from io import BytesIO
```

Comment:
I don't remember where this was written down, and it turns out it isn't actually cited on the [https://github.com/timothycrosley/isort](isort project page) like I thought it was, but the grouping I (and other people, apparently (I think)) is, in order: `__future__`, stdlib, third-party, intra-package.